backend/handler/FilterHandler.js

```diff
@@ -0,0 +1,39 @@
+const _ = require('lodash');
+const { db } = require('../db/Connection');
+
+const logger = require('./LoggerHandler').getLogger('FilterHandler');
+
+module.exports = (() => {
+    function getFilterUsage(filterId) {
+        return db.UserApp.findAll({ attributes: ['appData', 'username'] }).then(userApp => {
+            const result = [];
```

Comment:
We could make the variable name more concrete and provide a bit of context:

```suggestion
            const filterUses = [];
```

or

```suggestion
            const filterOccurrences = [];
```

Reply:
Done